packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -78,7 +78,7 @@ describe('HCP home page', () => {
     })
     checkHCPLayout(`${firstName} ${lastName}`)
     checkFooter()
-    checkPatientSecondaryBar()
+    checkPatientSecondaryBar(false, true)
```

Comment:
Maybe the method name has to be changed (chackSecondaryBar ?), because it's not only for Patient but also for caregiver and hcp ?